templates/new_home/index.html

```diff
@@ -134,30 +133,31 @@ <h2>MISSION</h2>
 <!-- About section ends -->
 
 <!--Initiatives Starts-->
-<div>
-    <center>
-        <h1 style="padding-top: 20px;" data-aos='fade-up'> Inititatives</h1>
-    </center>
-</div>
-
-<div class="card-container">
-    {% for initiative in initiatives %}
-    <div class="card" data-aos="fade-left" data-aos-duration="1500">
-        <div class="card-img">
-            <img style="height: 300px;" alt="image"
-                src="{% if initiative.thumbnail %}{{ initiative.thumbnail.url }}{% else %}{{ initiative.thumb_url }}{% endif %}">
-        </div>
-        <br>
-        <div class="card-text">
-            <h5 class="card-title">{{initiative.title}}</h5>
-            <p class="card-text">{{initiative.description}}</p>
+<section id="initiatives">
+    <div>
+        <center>
+            <h1 style="padding-top: 20px;" data-aos='fade-up'> Inititatives</h1>
+        </center>
```

Comment:
Instead of `center` tag, a `text-center` class can be added in the parent `div`.

Reply:
This was already there I just made a section around this.

Reply:
Yeah, I know that. But it'll be better if it also gets fixed here only.